bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElementModP.cs

```diff
@@ -214,18 +214,49 @@ private unsafe long[] GetNative()
             return data;
         }
 
+        public static bool operator ==(ElementModP a, ElementModP b)
```

Comment:
without this operator we had equality checks failing for these objects. i still feel like we should just compare the underlying data rathe than converting ot a hex string but I left it as implemented for now.